Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
 
 
 Compiler Features:
+ * Yul Optimizer: Rematerialize zero literals in order to take advantage of the PUSH0 opcode (introduced in Shanghai).
```

Comment:
We could describe this in a more high-level manner, e.g. something like:

```suggestion
 * Yul Optimizer: If ``PUSH0`` is supported, favour zero literals over storing zero values in variables.
```

But the current entry is also fine, if it quotes ``PUSH0``.